.cirrus.yml

```diff
@@ -131,42 +145,50 @@ task:
 
   build_image_script: |
     packer build \
-      -only "*.${CIRRUS_TASK_NAME}" \
+      -only "*.${IMAGE_NAME}" \
       -timestamp-ui \
       -force \
       -var "image_date=$DATE" \
-      -var "task_name=${CIRRUS_TASK_NAME}" \
+      -var "task_name=${IMAGE_NAME}" \
       -var gcp_project=$GCP_PROJECT \
       "$PACKERFILE"
     gcloud compute images add-iam-policy-binding \
       --project "$GCP_PROJECT" \
       --member=allAuthenticatedUsers \
       --role=roles/compute.imageUser \
-      pg-ci-${CIRRUS_TASK_NAME}-${DATE}
+      pg-ci-${IMAGE_NAME}-${DATE}
```

Comment:
It feels wrong that some of this references pg-ci-${IMAGE_NAME}, other parts just ${IMAGE_NAME}.

Reply:
Yes, that bothers me too. Maybe we can set global prefix environment variable and remove prefix from packer files? Then, we can set IMAGE_NAME something like ${PREFIX}-image_name.

Reply:
I like it - eventually we could use that to name images differently for PRs for example.